design-system/materials/internals/definition.yaml

```diff
@@ -383,6 +395,12 @@ decisionGroupsByTheme:
           choice: color-primary
         border-color-for-button-as-icon-when-disabled:
           choice: color-neutral
+        border-color-for-table-header:
+          choice: color-neutral
+        border-color-for-table-header-when-hovered:
+          choice: color-neutral
+        border-color-for-table-header-as-bottom:
+          choice: color-accent
```

Comment:
Sorry for the last name, but I couldn't think of anything better. Let me explain  this case a bit. In the new designs for each table header cell we have:
- a column divider
- a bottom border
<img width="418" alt="Screenshot 2022-11-30 at 13 20 45" src="https://user-images.githubusercontent.com/49066275/204794960-707ed0d3-5712-4d5f-bc52-6f5498c10745.png">
Both look the same for the new theme, but are different for the default theme. Therefore I introduced the `bottom` as variant here to differentiate the bottom border from other column divider.

As a side note, none of those tokens are really used to style the `border` css property 😅 `box-shadow` or `:before/:after` are used instead to style those "`borders`"